### PR TITLE
Explain how to build a local dev version

### PR DESCRIPTION
**Pull request by tobiasschweizer**
resolves https://dasch.myjetbrains.com/youtrack/issue/DSP-273

### CODE REVIEW

**Comment by tobiasschweizer** [comment]
@flavens Could you look at this if you find some time? I am preparing rc.1 and maybe it would be goodto include this PR.

File: README.md [hunk 1]
```diff
@@ -89,32 +101,31 @@ This package provides the following short-hand scripts:
...
 6. `npm run npm-publish`: Executes 4 and publishes the package to the NPM store (runs in dry-run mode).
 
-> Note: You need to install `yalc` globally by `npm install yalc -g` to use script number 3. In order to publish a package to NPM, you need to be logged in to NPM and you have to update the version in the `package.json`.
+> Note: You need to install `yalc` globally by `npm install yalc -g` to use script number 4.
 
-For further development with Knora the following scripts can be used:
+For further development with Knora, the following scripts can be used:
 
-1. `npm run integrate-admin-test-data <path-to-generated-client-code`: integrate generated test data for the Knora Admin API, 
+1. `npm run integrate-admin-test-data <path-to-generated-client-code`: integrates generated test data for the Knora Admin API,
```
Comment (flavens) [inline README.md:108]: missing `>`

File: README.md [hunk 2]
```diff
@@ -89,32 +101,31 @@ This package provides the following short-hand scripts:
 5. `npm run npm-pack`: Executes 1, 2 and packs the `./build/` folder into an NPM tgz package. The package is moved into a `./dist/` folder.
 6. `npm run npm-publish`: Executes 4 and publishes the package to the NPM store (runs in dry-run mode).
 
-> Note: You need to install `yalc` globally by `npm install yalc -g` to use script number 3. In order to publish a package to NPM, you need to be logged in to NPM and you have to update the version in the `package.json`.
+> Note: You need to install `yalc` globally by `npm install yalc -g` to use script number 4.
```
Comment (flavens) [inline README.md:104]: I would add a link to the yalc documentation somewhere

File: README.md (hunk 1 above)
Comment (tobiasschweizer) [reply README.md:108]: sorry, I don't understand what is missing.

File: README.md (hunk 2 above)
Comment (tobiasschweizer) [reply README.md]: done in https://github.com/dasch-swiss/knora-api-js-lib/pull/187/commits/327764f06f9d85786deeab6c9d2fad4f4aac09b7

File: README.md (hunk 1 above)
Comment (tobiasschweizer) [reply README.md:108]: ahh, I got it, finally :-)

Comment (tobiasschweizer) [reply README.md]: done in https://github.com/dasch-swiss/knora-api-js-lib/pull/187/commits/60b6c012982be1e668d8c5f103185e11a864e072

**Pull request merged**